src/App.jsx

```diff
@@ -1,19 +1,11 @@
-import About from "./components/organisms/About/About";
-import Gallery from "./components/organisms/Gallery/Gallery";
-import Footer from "./components/organisms/Footer/Footer";
-import Carousel from "./components/organisms/Carousel/Carousel";
-import TopNavigation from "./components/organisms/TopNavigation/TopNavigation";
+import Wrapper from "./components/organisms/Wrapper/Wrapper";
```

Comment:
Why do you have double `Wrapper` in the path?

Reply:
It's Folder/File.jsx 
I know that you wrote about this in MS Teams, but as I understood this is just another approach for calling components. 
Is it possible to leave it that way for this task ?

Reply:
Done